README.md

```diff
@@ -52,3 +52,36 @@ Running Tests
 - To run the simple http request tests, first click the ResponseTestSuite link, then click the Suite button.
 - To run the tests that require threading, first click the SimultaniousTestSuite link, then click the Suite button.
 - To run a test individually, first click on the test link, then click the Test button.
+
+Adding CI
+------------
+- Fork cob spec
+- On your local machine navigate to your server's directory
+- Create a submodule using the forked copy of cob spec `git submodule add https://github.com/myusername/cob_spec`
+- Within this new submodule navigate to `FitNesseRoot` and create a new directory `mkdir PassingTests`
+- Copy contents of `HttpTestSuite` into your new directory `PassingTests` by issuing the following command: `cp -r HttpTestSuite/ PassingTests/`
+- Navigate into `PassingTests/ResponseTestSuite` and add/remove cob spec tests as necessary
+- To add a test to the `PassingTests`: `cp -r HttpTestSuite/ResponseTestSuite/SimpleGet PassingTests/ResponseTestSuite/SimpleGet/`
```

Comment:
I was under the impression that there is a way to include a test in a different suite without having to duplicate it. Possibly by using something like [Fitnesse's Include feature](http://fitnesse.org/FitNesse.FullReferenceGuide.UserGuide.FitNesseWiki.MarkupLanguageReference.MarkupPageInclude)

There's a chance I am wrong about this though.